src/Queue/Client.php

```diff
@@ -31,11 +33,24 @@ class Client extends BaseClient
      */
     protected static $httpClient;
 
+     /**
+      * Attribute to track HTTTP Client used for sending requests
+      *
+      * @var NS8HttpClient $ns8HttpClient
```

Comment:
You don't need to specify the variable name here. `@var NS8HttpClient` is sufficient.